src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -64,6 +92,15 @@ public void setNeutral() {
     leftMotorFollower.setControl(new NeutralOut());
   }
 
+  public void setVoltage(Voltage voltage) {
+    rightMotorLeader.setControl(voltageRequest.withOutput(voltage.in(Units.Volts)));
+  }
+
+  public void setSoftwareLimits(boolean reverseLimitEnable, boolean forwardLimitEnable) {
+    constElevator.ELEVATOR_CONFIG.SoftwareLimitSwitch.ReverseSoftLimitEnable = reverseLimitEnable;
+    constElevator.ELEVATOR_CONFIG.SoftwareLimitSwitch.ForwardSoftLimitEnable = forwardLimitEnable;
+  }
```

Comment:
This **needs to be called before zeroing begins**. basically telling the robot to ignore the software position limits as it may bypass them when zeroing (if the zeroing was incorrect to start).

This also needs to reapply the config so that the motors actually change.

add
`rightMotorLeader.getConfigurator().apply(constElevator.ELEVATOR_CONFIG);`
but do it for both motors